library/core/src/cmp.rs

```diff
@@ -676,12 +688,19 @@ impl<T: Clone> Clone for Reverse<T> {
 ///
 /// ## Corollaries
 ///
-/// From the above and the requirements of `PartialOrd`, it follows that `<` defines a strict total order.
-/// This means that for all `a`, `b` and `c`:
+/// From the above and the requirements of `PartialOrd`, it follows that for
+/// all `a`, `b` and `c`:
 ///
 /// - exactly one of `a < b`, `a == b` or `a > b` is true; and
 /// - `<` is transitive: `a < b` and `b < c` implies `a < c`. The same must hold for both `==` and `>`.
 ///
+/// Mathematicaly speaking, the `<` operator defines a strict [weak order]. In
```

Comment:
Oh, almost missed this. Typo.

```suggestion
/// Mathematically speaking, the `<` operator defines a strict [weak order]. In
```

Reply:
Corrected with fixup 12ed8a8cbd38e7aa262d2cc3198f952d5f23cbbf.